sources/osgDB/DatabasePager.js

```diff
@@ -431,6 +425,21 @@ MACROUTILS.createPrototypeObject(
             });
             availableTime -= elapsedTime;
             return availableTime;
+        },
+
+        // Helper functions to avoid undesired memory allocation
+        _sortByTimeStamp: function(r1, r2) {
```

Comment:
thoses functions could even be outside of the object, unless the user could overwrite it. It could be full static function

Reply:
yeah the point is letting the user to implement it's own priority functions